<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formAdmin</class>
 <widget class="QDialog" name="formAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>786</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabAdmin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1331</width>
     <height>731</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Buscar</string>
    </attribute>
    <widget class="QTextEdit" name="infouser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>571</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Heavy</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="correoText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarA">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ingrese un Correo: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="in">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>210</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>InOrden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="post">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>260</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PostOrden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>310</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PreOrden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>170</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Usuarios:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Carga Masiva</string>
    </attribute>
    <widget class="QPushButton" name="cargarU">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Usuarios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cargarP">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Publicaciones</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Solicitudes</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="jsonText">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>441</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Reportes</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Grafica Lista doble</string>
     </property>
    </widget>
    <widget class="QLabel" name="reportes">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>401</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Graficar Arbol AVL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>80</y>
       <width>611</width>
       <height>551</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Graph">
    <attribute name="title">
     <string>Grafo</string>
    </attribute>
    <widget class="QLabel" name="label_ady">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>491</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargando...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_g">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>100</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargando...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Lista de Adyacencia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Grafo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="salirAdmin">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
